video/profiles.go

```diff
@@ -217,6 +236,10 @@ type EncodedProfile struct {
 	ColorDepth   int64  `json:"colorDepth,omitempty"`
 	ChromaFormat int64  `json:"chromaFormat,omitempty"`
 	Quality      uint   `json:"quality,omitempty"`
+	// Copy is a flag to indicate that the profile should be a copy of the input video, no transcoding required. Copying
+	// cannot be specified externally, but is automatically set when the input is in HLS format. This field is not
+	// supported on broadcasters trancode request, so should be used only for internal logic.
+	Copy bool `json:"-"`
```

Comment:
If we leave it blank it will still be serialized when calling the broadcaster and it would likely explode cause the field doesn't exist (unless it doesn't validate that, but it's still wrong to send a non-existent field IMO).

We could use an `omitempty` instead, given we never send any `Copy: true` values to broadcasters (given that validation I added), but it feels unclear IMO. So `-` makes it clear that this field is never to be serialized (it's go's `json` way to ignore a field).

Reply:
Let's continue this discussion on https://github.com/livepeer/catalyst-api/pull/1261#discussion_r1607518382